prod.dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:12.16.3-alpine as build
+FROM node:16.13.0-alpine as build
```

Comment:
AWS still does not have node 16 support, please change it to 14

Reply:
Done ✅